src/snarky.d.ts

```diff
@@ -155,8 +155,109 @@ declare class Field {
    */
   toFields(): Field[];
 
+  /**
+   *
+   * Check if this {@link Field} is lower than another Field-like value.
+   * Returns a {@link Bool}.
+   *
+   * ```ts
+   * Field(2).lessThan(3); // Bool(true)
+   * ```
+   */
+  lessThan(y: Field | number | string | boolean): Bool;
+  /**
+   *
+   * Check if this {@link Field} is lower than or equal to another Field-like value.
+   * Returns a {@link Bool}.
+   *
+   * ```ts
+   * Field(2).lessThanOrEqual(3); // Bool(true)
+   * ```
+   */
+  lessThanOrEqual(y: Field | number | string | boolean): Bool;
+  /**
+   *
+   * Check if this {@link Field} is greater than another Field-like value.
+   * Returns a {@link Bool}.
+   *
+   * ```ts
+   * Field(2).greaterThan(1); // Bool(true)
+   * ```
+   */
+  greaterThan(y: Field | number | string | boolean): Bool;
+  /**
+   *
+   * Check if this {@link Field} is greater than or equal to another Field-like value.
+   * Returns a {@link Bool}.
+   *
+   * ```ts
+   * Field(2).greaterThanOrEqual(1); // Bool(true)
+   * ```
+   */
+  greaterThanOrEqual(y: Field | number | string | boolean): Bool;
+
   // TODO: Make these long form version
   /**
+   *
+   * Assert that this {@link Field} is lower than another Field-like value.
+   *
+   * ```ts
+   * Field(1).assertLessThan(2);
+   * ```
+   *
+   * This function can only be called inside a checked computation, like a
+   * SmartContract method, and causes it to fail if the assertion fails.
```

Comment:
Didn't we fix that this "can only run in a checked computation"? Should remove those comments here and elsewhere

Reply:
Oh ok, well I hope we can merge it to main soon